public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+    
+    The component lifecycle has an explicit point when the content of `HeroJobBoard` is collected by the framework, and then `heroesToInvite` is updated. You can hook into
+    this lifecycle event by defining the `ngAfterContentInit()` method, and carry out any additional setup you may need to use the collection of children.
+
+:marked
+  We do not need to change any other components to complete our task. When the app is running, and the **Invite heroes** button is clicked, we 
+  can see 8 heroes instead of the 6 we saw in the previous scenarios:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/with-contentchildren-ui.png" alt="Heroes by @ContentChildren")
+  
++makeExample('component-communication/ts/fragments/heroes-to-invite-2.ts', null, 'take-job-contentchildren/hero-job-board.ts (extract)', {blk: /(@ContentChildren\(HeroCrm\)|QueryList&lt;HeroCrm&gt;)/g, pnk: /(heroesToInvite)/g})
+
+:marked
+  The `JobBoardWithContent` component nests 8 `hero-crm` children: 6 coming from `Hero.heroes`, and 2 others (Dynama and Magma) added with direct markup.
+  
+  ### A few additional comments about `@ContentChildren`
+  
+  * `@ContentChildren` provides a mechanism for a parent to collect its children defined as content. We can use _all the communication techniques between the
+  parent and its children_ (in both direction) you already learned in this chapter.
+  * Components can have multiple `@ContentChildren` properties witch different child types.
+  * `QueryList` has a `changes` property that provides a `subscribes()` method to be notified about changes in the content children.
+  
+  ## Conclusion
+  
+  Angular provides great communication mechanisms among components within an application. If we have components within the same tree or subtree,
+  we have parent and child components, and can use these methods:
+  * **Parent to child**: data binding with using one or more `@Input()` propery in the child component, the values of which can be assigned in the parent's 
+  template.
+  * **Child to parent**: event binding with using one or more `@Output()` properties (`EventEmitter` instances) in the child component, the event handlers 
+  of which can be assigned in the parent's template.
+  
+  If we have have unrelated components that are in separate subtrees, they can communicate **through intermediary components**, such as services:
+  * The same intermediary component instance can be injected into the the communicating parties. 
+  * The parties utilize the intermediary component, 
+    * they can access its properties, 
+    * invoke its methods,
+    * and subscribe to its events.
+    
+  As we discussed, intermediary components are great alternatives for parent to child, and child to parent communication, too. With the help of 
+  `@ContentChildren` we can transpose children into their parent, and utilize the parent-to-child or child-to-parent communication techniques we 
```

Comment:
The component children case should be in a new paragraph as it has nothing to do with "intermediary components"